### PR TITLE
[GNA] Apply Style formatter

### PR DESCRIPTION
**Pull request by marcinkusm**
### Details:
 - enable clang-format for gna plugin and gna plugin tests targets
 - apply formatter
 - fix compilation issues with includes after reformatting

### Tickets:
 - 102474

### CODE REVIEW

File: src/plugins/intel_gna/legacy/include/legacy/details/ie_cnn_network_tools.h [hunk 1]
```diff
@@ -4,16 +4,16 @@
 
 /**
  * @brief A header file for CNNNetwork tools
- * 
+ *
  * @file ie_cnn_network_tools.h
  */
 #pragma once
+#include <legacy/ie_layers.h>
```
Comment (ilya-lavrenov) [inline src/plugins/intel_gna/legacy/include/legacy/details/ie_cnn_network_tools.h:11]: All OpenVINO headers should be included via `""`

Comment (marcinkusm) [reply src/plugins/intel_gna/legacy/include/legacy/details/ie_cnn_network_tools.h]: Thanks for comment.
This is results of applying of automatic fix jobs for clan-format, this is just reorder of existing includes.
Fix of includes types would be done in separate tasks.

**Review by sirzabek — COMMENTED** [review]
Why are there changes related to including headers?

**Comment by marcinkusm** [comment]
Applying style format reorganize includes to group them based on include types
`""` and system "<>".
It looks like some includes where missing but, code was compiling due the fact in the places there were used another headers containing missing declaration-s were included.

**Review by sirzabek — APPROVED** [review]

**Review by mryzhov — APPROVED** [review]

**Pull request merged**